src/services/__tests__/userServices.test.js

```diff
@@ -19,4 +20,27 @@ describe('UserServices', () => {
       done();
     });
   });
+
+  it('should return default locale if no locale cookie is present', done => {
+    userServices.getLocale().then(locale => {
+      expect(locale).toMatchSnapshot();
+      done();
+    });
+  });
+
+  it('should return en locale if cookie is set to en_US', done => {
+    Cookies.get = jest.fn().mockImplementation(() => 'en_US');
+    userServices.getLocale().then(locale => {
+      expect(locale).toMatchSnapshot();
+      done();
+    });
+  });
+
+  it('should return test locale if cookie is set to test_US', done => {
```

Comment:
request: lets change the description on this one tooooo: `should return the default locale with an invalid ISO_639 code`